sys.py/UI/skin_manager.py

```diff
@@ -23,52 +23,57 @@ class SkinManager(object):
     _Config = None
     
     def __init__(self):
-        pass
+        self.Init()
 
     def ConvertToRGB(self,hexstr):
         
         h = hexstr.lstrip('#')
         return tuple(int(h[i:i+2], 16) for i in (0, 2 ,4))
     
     def Init(self):
-        
+        if not SkinManager._Colors:
+            self.SetColors()
+
+    def SetColors(self):
         Colors = {}
-        Colors["High"] = pygame.Color(51,166,255) 
-        Colors["Text"] = pygame.Color(83,83,83)
-        Colors["Front"] =  pygame.Color(131,199,219)
-        Colors["URL"]   = pygame.Color(51,166,255)
-        Colors["Line"]  =  pygame.Color(169,169,169)
-        Colors["TitleBg"] = pygame.Color(228,228,228)
-        Colors["Active"]  =  pygame.Color(175,90,0)
-        Colors["White"]  = pygame.Color(255,255,255)
-        
-        self._Colors = Colors
-        
+        Colors["High"] = pygame.Color(51, 166, 255)
+        Colors["Text"] = pygame.Color(83, 83, 83)
+        Colors["Front"] = pygame.Color(131, 199, 219)
+        Colors["URL"] = pygame.Color(51, 166, 255)
+        Colors["Line"] = pygame.Color(169, 169, 169)
+        Colors["TitleBg"] = pygame.Color(228, 228, 228)
+        Colors["Active"] = pygame.Color(175, 90, 0)
+        Colors["Disabled"] = pygame.Color(204, 204, 204)
```

Comment:
I added "Disabled" when I saw 204,204,204 used multiple times in the UI for when no elements in a list are displayed.

If you want, we can rename it "Grey" or something else.